lstchain/image/muon/muon_analysis.py

```diff
@@ -221,17 +195,17 @@ def analyze_muon_event(subarray, tel_id, event_id, image, good_ring_config, plot
 
     tel_description = subarray.tels[tel_id]
 
-    cam_rad = (
-                      tel_description.camera.geometry.guess_radius() / tel_description.optics.equivalent_focal_length
-              ) * u.rad
     geom = tel_description.camera.geometry
-    equivalent_focal_length = tel_description.optics.equivalent_focal_length
+    geom_telframe = geom.transform_to(TelescopeFrame())
+
+    fov_rad = geom_telframe.guess_radius()
+
     mirror_area = tel_description.optics.mirror_area
 
     # some parameters for analysis and cuts for good ring selection:
     params = update_parameters(good_ring_config, geom.n_pixels)
 
-    x, y = pixel_coords_to_telescope(geom, equivalent_focal_length)
+    x, y = geom_telframe.pix_x, geom_telframe.pix_y
     muonringparam, clean_mask, dist, image_clean = fit_muon(x, y, image, geom)
```

Comment:
Why does this function get the untransformed camera geometry? Why does it get the coordinates and the geometry?